_topic_map.yml

```diff
@@ -674,6 +674,9 @@ Topics:
 - Name: Kuryr SDN Administration
   File: kuryr
   Distros: openshift-origin
+- Name: Disabling Features using Feature Gates
+  File: disabling_features
+  Distros: openshift-origin,openshift-enterprise,openshift-dedicated
```

Comment:
@spurtell - would this feature be available in Dedicated docs at any point? 

@mburke5678

Reply:
There are no plans to make this available in OpenShift Dedicated.

Reply:
Removed dedicated from topic map.
https://github.com/openshift/openshift-docs/pull/9856